src/improvements/tasks/eth/001-opcm-upgrade-v200/README.md

```diff
@@ -0,0 +1,28 @@
+# 001-opcm-upgrade-v200: Mainnet OPCM v2.0.0: Base
+
+Status: [DRAFT]()
+
+Executes [Upgrade 13](https://gov.optimism.io/t/upgrade-proposal-13-opcm-and-incident-response-improvements/9739) on Base Mainnet.
+
+In summary, this task uses `op-contract/v2.0.0` [OPContractsManager](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v2.0.0-rc.1/packages/contracts-bedrock/src/L1/OPContractsManager.sol) to upgrade 1 chain:
+
+1. Base Mainnet
+
+### Timing
+
+Expected to be executed on or around 2025-04-02.
+
+## Transaction creation
+
+The transaction is generated by the [OPCMV200 template script](../../../template/OPCMUpgradeV200.sol),
+which reads the inputs from the [`config.toml`](./config.toml) fil
+
+## Signing and execution
+
+When simulating, ensure the logs say `Using script <your_path_to_superchain_ops>/superchain-ops/src/improvements/template/OPCMUpgradeV200.sol`.
+Navigate to the correct task directory then run the simulate command.
+
+```
+cd 001-opcm-upgrade-v200
```

Comment:
Would be useful to add all simulate commands here for clarity like I did with unichain: 

https://github.com/ethereum-optimism/superchain-ops/pull/784/files#diff-5d82b0078b2b7ab533502b56fe9e16b28f6ddd609c6d1551de37466bb1e67579R36-R42